src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -44,8 +44,9 @@ void HexagonGame::initLua_Utils()
     });
     lua.writeVariable("u_isKeyPressed",
         [this](int mKey) { return window.getInputState()[KKey(mKey)]; });
-    lua.writeVariable(
-        "u_haltTime", [this](float mDuration) { status.timeStop = mDuration; });
+    lua.writeVariable("u_haltTime", [this](float mDuration) {
+        status.pauseTime(ssvu::getSecondsToFT(mDuration));
+    });
```

Comment:
You are doing the exact opposite of what I've told you to do. You are MULTIPLYING it by the framerate, not DIVIDING it. Use getFTToSeconds to convert frame time to seconds.